docs/application/native/guides/app-management/cion.md

```diff
@@ -0,0 +1,235 @@
+    ret = cion_client_connect(client, peer_info);
+    ```
+
+    > [!NOTE]
+    > You can get the `peer_info` from discovered callback function.
+
+    > [!NOTE]
+    > You can add callback function using `cion_client_add_connection_result_cb` to get the connection result.
+
+4. Send the data:
+
+    ```cpp
+    unsigned char str[] = "test_data";
+    unsigned char *return_data = NULL,
+    unsigned int return_size;
+
+    ret = cion_client_send_data(client, strlen((const char*)str), 5000, &return_data, &return_size);
+    ```
+5. Send the payload:
+
+    ```cpp
+    static void __cion_client_payload_async_result_cb(
+        const cion_payload_async_result_h result, void *user_data);
+    ```
+    ```cpp
+    cion_payload_h payload;
+    unsigned char data[] = "test_payload";
+
+    ret = cion_payload_create(&payload, CION_PAYLOAD_TYPE_DATA);
+    ret = cion_payload_set_data(payload, data, sizeof(data));
+
+    ret = cion_client_send_payload_async(client, payload, __cion_client_payload_async_result_cb, NULL);
+    ```
+
+Cion server can listen the request from the client. If a connection request comes from the client, the server can accept connection.
+After connection is accepted, the server can receive data and payload.
+
+Cion server can be used as follows:
+
+1. Create the server handle:
+
+    ```cpp
+    cion_server_h server;
+
+    ret = cion_server_create(&server, "TEST_SERVICE", "MY_DISPLAY_NAME", NULL);
+    ```
+
+2. Server is ready to listen the requests from the client:
+
+    ```cpp
+    static void __cion_server_connection_request_cb(const char *service_name,
+        const cion_peer_info_h peer_info, void *user_data) 
+    {
+    }
+    ```
+    ```cpp
+    int cion_server_listen(server, __cion_server_connection_request_cb, void *user_data);
+    ```
+
+3. Accept the connection:
+
+    ```cpp
+    ret = cion_server_accept(server, peer_info);
+    ```
+
+    > [!NOTE]
+    > You can get the `peer_info` from connection request callback function.
+
+4. Add the callback function to receive data and payload:
+
+    ```cpp
+    static void __server_data_received_cb(const char *service_name,
+	    const cion_peer_info_h peer_info, const unsigned char *data,
+		unsigned int data_size, unsigned char **return_data,
+		unsigned int *return_data_size, void *user_data)
+    {
+    }
+    ```
+    ```cpp
+    ret = cion_server_set_data_received_cb(server, __server_data_received_cb, NULL);
+    ```
+
+    ```cpp
+    static void __server_payload_received_cb(const char *service_name,
+		const cion_peer_info_h peer_info, const cion_payload_h payload,
+		cion_payload_transfer_status_e status,
+		void *user_data)
+    {
+    }
+    ```
+    ```cpp
+    ret = cion_server_add_payload_received_cb(server, __server_payload_received_cb, NULL);
+    ```
+
+<a name="group"></a>
+## Communicate in group style
+
+Cion API provides functions to communicate with the other applications in group style.
+
+Cion group can subscribe topic and publish data.
```

Comment:
Line 169: The meaning is not clear, especially the part "...can subscribe topic and publish data"
@sukhyungkang would you please rephrase the sentence? Alternatively, you may explain/elaborate more on what you are trying to say here so I may offer a suggestion...

Reply:
it's changed